data/_demodata/applications/datacatalogue/Mappings.csv

```diff
@@ -1 +0,0 @@
-source,source version,target,target version,cdms other,mapping status,ETL frequency,ETL specification url,ETL specification document
```

Comment:
this file should be deleted?

Reply:
It is already deleted.... strange 
![image](https://github.com/user-attachments/assets/e3f552e2-026a-4f86-9d8a-7cadae57d878)